covid19_call_variants.sh

```diff
@@ -108,8 +108,20 @@ bcftools view \
 # in the special indexed gzip format (can't use regular gzip)
 bcftools index "${prefix}.vcf.gz"
 
+# We want to generate a consensus sequence with:
+# 1. well-supported deletions marked with '-'
+# 2. low-coverage regions masked with N
+
+# vcf is 1-based while bedgraph is 0-based; thus the somewhat convoluted path to mask.bed
+bcftools query -f'%CHROM\t%POS0\t%END\n' ${prefix}.vcf.gz > variants.bed
+# get low coverage sites (<10) in bedgraph format
+bedtools genomecov -bga -ibam ${prefix}.sorted.bam | awk '$4 < 10' > low_coverage_sites.bed
```

Comment:
@christinehe What do you think about exposing the value that defines low coverage as a parameter?

Reply:
+1 this will make it easy to generate a `consensus.fasta` at a bunch of different min depths.

Reply:
Resolved in [e2754e6](https://github.com/onecodex/sars-cov-2/pull/57/commits/e2754e6893a8bc80af15dc425ba248a7e90d0dc9)